cmd/helpMessages.go

```diff
@@ -129,7 +129,7 @@ Download a subset of containers within a storage account by using a wildcard sym
 
   - azcopy cp "https://[srcaccount].blob.core.windows.net/[container*name]" "/path/to/dir" --recursive
 
-Download all the versions of a blob from Azure Storage to local directory. Ensure that source is a valid blob, destination is a local folder and versionidsFile which takes in a path to the file where each version is written on a separate line. All the specified versions will get downloaded in the destination folder specified.
+Download all the versions of a blob from Azure Storage to local directory listed in a text file (i.e, versionidsFile). Ensure that source is a valid blob, destination is a local folder and versionidsFile is a text file where each version is written on a separate line. All the specified versions will get downloaded in the destination folder specified.
```

Comment:
From https://github.com/Azure/azure-storage-azcopy/wiki/Listing-specific-files-to-transfer:

>> The reason that it's undocumented is that, at high file counts, it has slow performance for downloads and service-to-service copies. We want to fix that before we document it more officially

We should think about whether we want to peel this text back as the param --list-of-files is marked hidden for copy commands